crates/volta-core/src/platform/mod.rs

```diff
@@ -16,7 +16,7 @@ pub mod sourced;
 pub use self::sourced::{Source, SourcedImage, SourcedPlatformSpec};
 
 /// A specification of tool versions needed for a platform
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
That's a reasonable thought, and sort of where I was leading. My one question is does the `derive`d implementation depend on the order that the fields are declared in within the struct? I would imagine it has to, and given that we should probably have a comment that makes it clear not to re-order them, since we are relying on the ordering being first by name, then by version.

Reply:
I'm actually not positive. We can dig in, for sure; it's worth an investigation.